src/Grid/IColumnModelOptions.ts

```diff
@@ -3,7 +3,6 @@ import { AggregateFunctions, ColumnDataType, ColumnSortDirection } from './Colum
 export default interface IColumnModelOptions {
     Aggregate?: AggregateFunctions;
     DataType?: ColumnDataType;
-    Filter?: object;
     Filtering?: boolean;
```

Comment:
Do we need this?

Reply:
Yes

Reply:
Yes, it's necessary to set the filter properties for the columns